pkg/api/server/register_containers.go

```diff
@@ -126,11 +147,10 @@ func (s *APIServer) RegisterContainersHandlers(r *mux.Router) error {
 	//   '500':
 	//     "$ref": "#/responses/InternalError"
 	r.HandleFunc(VersionedPath("/containers/{name:..*}/json"), APIHandler(s.Context, generic.GetContainer)).Methods(http.MethodGet)
-	// swagger:operation POST /containers/{nameOrID}/kill containers killContainer
-	//
-	// Kill Container
-	//
+	// swagger:operation post /containers/{nameorid}/kill containers killcontainer
 	// ---
+	// summary: Kill container
+	// description: send a posix signal to a container, defaulting to killing to the container.
```

Comment:
For the `signal` parameter: `Signal to send to the container as an integer or string (e.g. SIGINT)`